lib/improver/utilities/cube_metadata.py

```diff
@@ -145,11 +145,23 @@ def add_coord(cube, coord_name, changes, warnings_on=False):
         bounds = changes['bounds']
     units = None
     if 'units' in changes:
-        units = changes["units"]
-    new_coord = new_coord_method(long_name=coord_name,
-                                 points=points,
-                                 bounds=bounds,
-                                 units=units)
+        units = changes['units']
+    var_name = None
+    if 'var_name' in changes:
+        var_name = changes['var_name']
+
+    try:
+        new_coord = new_coord_method(
+            standard_name=coord_name, var_name=var_name, points=points,
```

Comment:
Can someone add all 3 names? `standard_name`,  `long_name` and `var_name`? Maybe we need to explain which combinations are valid in the docstring?

Reply:
This design mirrors how iris is set up: https://scitools.org.uk/iris/docs/v1.9.0/html/iris/iris/coords.html?highlight=coord%20rename#iris.coords.Coord.rename  If a standard name is provided then the standard name is set; otherwise the long name is set.  The var name can be set independently of these two.

Reply:
Ok, that's fine